src/PostList.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Lullabot\Parsely;
+
+/**
+ * Class PostList.
+ *
+ * @todo Do all lists of any object have the same format? If so, this could
+ *   become just 'List'.
+ */
+class PostList implements \ArrayAccess, \Countable
```

Comment:
This exists so we can extend it later to support transparent paging. That may mean we have to scale this back to `\Iterator`, but the idea is if you have a post list of 10000 results with a limit set set to 50, every time you `foreach` over a page boundary we can transparently fetch it.